README.md

```diff
@@ -1,6 +1,6 @@
 # ONNX Runtime GenAI
 
-## *Main branch contains new API changes and examples in main branch reflect these changes. For example scripts compatible with current release (0.5.2), [see release branch](https://github.com/microsoft/onnxruntime-genai/tree/rel-0.5.2).*
+## *Main branch contains new API changes and examples in main branch reflect these changes. For example scripts compatible with current release (0.7.0), [see release notes of release branch](https://github.com/microsoft/onnxruntime-genai/tree/rel-0.7.0).*
```

Comment:
```suggestion
Note: between release candidate 0.7.0-rc2 and release 0.7.0 there is a breaking Python API change in `tokenizer.encode(prompt)`. Previously this method returned a Python list and now returns a numpy array. When concatenating the tokens generated by two prompts to pass to `append_tokens` e.g a system prompt and a user prompt, you must use the following instead of `system_prompt + input_tokens`:

```python
system_tokens = tokenizer.encode(system_prompt)
input_tokens = tokenizer.encode(prompt)
generator.append_tokens(np.concatenate([system_tokens, input_tokens]))
```